app/javascript/styles/mastodon/components.scss

```
@@ -7419,6 +7419,13 @@ noscript {
           span {
             user-select: all;
           }
+
+          .icon-lock {
+            height: 16px;
+            width: 16px;
+            position: relative;
+            top: 3px;
```

imho the following gives a little better alignment, but that's only my own opinion and both are an improvement:
```suggestion
            vertical-align: text-top;
```

In cases like this we use icons sizes far below what they were designed for, but I'm not sure what else we can do.